extensions/kafka-streams/deployment/src/main/resources/dev-templates/kafka-streams-topology.html

```diff
@@ -1,14 +1,49 @@
 {#include main fluid=true}
 {#style}
+svg {
```

Comment:
I think you should have an enclosing class like `.topology` and make all the very generic elements `.topology element`. Typically for `.caret`.

Reply:
Fixed. 
I've introduced an enclosing div with `id="kafka-streams-topology"`. I use this id to limit the CSS blast radius. 
Maybe this should be a documented practice for extension developers?